trulens_eval/examples/snowflake_demo/.env.template

```diff
@@ -2,11 +2,11 @@ REPLICATE_API_TOKEN=
 PINECONE_API_KEY=
 
 # Snowflake details
-SF_USER=
```

Comment:
Renamed env vars since trulens cortex provider uses `SNOWFLAKE_*` by default and didn't want to duplicate these keys